configs/config.go

```diff
@@ -81,22 +81,22 @@ type RedisConfig struct {
 	DB       int    `mapstructure:"db"`
 }
 
-type RPCBatchSizeConfig struct {
+type RPCBatchRequestConfig struct {
 	BlocksPerRequest int `mapstructure:"blocksPerRequest"`
 	BatchDelay       int `mapstructure:"batchDelay"`
 }
 
-type RPCTracesConfig struct {
```

Comment:
where did the Trace config go?

Reply:
afaiu it's now `ToggleableRPCBatchRequestConfig` and reused for both `BlockReceipts` and `Traces`

Reply:
yes, since traces and block receipts have the same config, decided to generalize this